templates/unix-qa-stage.yml

```diff
@@ -16,6 +16,17 @@ stages:
           clean: all
         displayName: "Run UTs"
         steps:
+          - task: DotNetCoreCLI@2
+            displayName: 'Build test pre-requisites'
+            inputs:
+              command: 'publish'
+              projects: './Tests/LogArgs/LogArgs.csproj'
+              publishWebProjects: false
+              zipAfterPublish: false
```

Comment:
I suspect that `publishWebProjects` and `zipAfterPublish` were the missing parts, right?